README.md

```diff
@@ -67,14 +72,14 @@ The configuration file should contain the following "JSON-SNIPPET".
 
 The API endpoint should cover the following steps:
 
-1. Fetch the current Email object by the given ID
-1. After fetching it, send the email and process the result in the following way:
-    - When the email delivery was successful
-      - Delete current Email object
-      - Set the Http status code "200" to the response
-    - When the email delivery fails temporarily
-      - Set the status of the Email object to "pending"
-      - Set the Http status code "503" to the response
-    - When the email delivery fails permanently.
-      - Set the status of the Email object to "error"
-      - Set the Http status code "400" to the response
+1. Fetch the current e-mail object by the given id (commercetools CustomObject UUID)
+1. After fetching it, send the e-mail and process the result in the following way:
+    - If the e-mail delivery is successful
+      - Delete the current e-mail object
+      - Set the response Http status code to `200`
+    - If the e-mail delivery fails temporarily (TODO: WHAT DOES IT MEAN)
```

Comment:
Hi @ahalberkamp could you please help me here describe what temporarily and permanently mean exactly? Thanks.